### PR TITLE
Update User Guide's ingredient viewing and resetting features

### PR DESCRIPTION
**Pull request by WM71811**

### CODE REVIEW

**Review by Diwu-Yi — DISMISSED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -308,9 +309,9 @@ Action | Format, Examples
 **Set**  | `set INGREDIENT_NAME AMOUNT` <br> e.g., `set milk 20`
 **Set all**  | `set all ingredients AMOUNT` <br> e.g., `set all ingredients 5`
 **Set ingredients**  | `set ingredients m/MILK b/BLACK_TEA s/SUGAR o/BUBBLE` <br> e.g., `set ingredients m/19 b/20 s/5 o/0`
-**Get**| `ingredient single INGREDIENT_NAME`  <br> e.g., `ingredient single milk`
-**Reset**| `ingredient resetAll`
-**List**| `ingredient list`
+**Get**| `i-view-single INGREDIENT_NAME`  <br> e.g., `i-view-single i/Milk`
```
Comment (ureshiiYing) [inline docs/UserGuide.md]: Perhaps it is better to say **View single ingredient**?

**Review by yanbingtao — DISMISSED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (WM71811) [reply docs/UserGuide.md]: noted with thanks

**Review by yanbingtao — APPROVED** [review]
LGTM

**Review by Diwu-Yi — APPROVED** [review]
LGTM

**Pull request merged**